docs/source/teams/cloud_media.rst

```diff
@@ -37,6 +42,76 @@ You can get information about currently cached media files for a sample collecti
     print(foc.media_cache.stats())
     foc.media_cache.clear()
 
+
+Some other convenient methods provided in the FiftyOne Teams Python SDK that
+can be used to manipulate both local and cloud media include:
+
+* ``fiftyone.core.storage.copy_files(inpaths,outpaths,skip_failure=False,progress=True)``
+
+* ``fiftyone.core.storage.move_files(inpaths,outpaths,skip_failure=False,progress=True)``
+
+* ``fiftyone.core.storage.delete_files(paths,skip_failure=False,progress=True)``
+
+* ``fiftyone.core.storage.list_files(dirpath,abs_paths=False,recursive=False,include_hidden_files=False,sort=True)``
+
+* ``fiftyone.core.storage.upload_media(sample_collection,remote_dir,rel_dir=None,media_field="filepath",update_filepaths=False,overwrite=False,skip_failures=False,progress=False)``
+
+
+.. note::
+
+   Pro tip: Learn more about these and other FiftyOne methods by entering them
+   into an IPython cell followed by a question mark:
+
+   .. code-block:: python
+
+        import fiftyone.core.storage as fos
+
+        fos.upload_media?
+
+   .. code-block:: shell
```

Comment:
The docs build also complains for me here, I think because of the '``' in the block.